package.json

```diff
@@ -64,6 +64,7 @@
     "ycsb": "node ./benchmark/ycsb.js  run -P ./benchmark/workloada -p table=usertable -p cloudspanner.instance=ycsb-instance -p operationcount=100 -p cloudspanner.database=ycsb"
   },
   "dependencies": {
+    "@google-cloud/common": "^0.23.0",
```

Comment:
fun fact - you don't need to do that!  This is available on `common-grpc` in the `util` import as well :)